--- Expr.hs~	2008-12-22 21:24:39.000000000 -0800
+++ Expr.hs	2008-12-22 22:21:56.000000000 -0800
@@ -27,6 +27,8 @@
 
 data Operator tok st a   = Infix  (GenParser tok st (a -> a -> a)) Assoc
                          | Prefix (GenParser tok st (a -> a))
+-- NOTE kramer@acm.org reto -- the Postfix arm was missing in the compat module
+                         | Postfix (GenParser tok st (a -> a))
 
 type OperatorTable tok st a = [[Operator tok st a]]
 
